mods/persistence/modules/mob/observer/limbo.dm

```diff
@@ -2,22 +2,41 @@
 // still floating around .. somewhere.
 /mob/living/limbo
 	use_me = 0
+	stat = UNCONSCIOUS
 
-/mob/living/limbo/Initialize()
-	. = ..()
+	var/obj/item/organ/internal/stack/cortical_stack
```

Comment:
You sure it's a good idea to keep a ref directly like that? Because, if it has a stack it'll be iniside of it, no?
loc references don't prevent deleting, but that var would.

Reply:
Sure, the cortical stack should clear this on deletion.